0123-redefining-data-only-onboarding-and-assertion-fees.md

```diff
@@ -11,78 +11,74 @@
-    - Developers of Helium Wallets or Maker Apps that allow location assertion
-    - Developers of Helium CLI and API tools that allow location assertion
+| Action                                            | Cost               |
+| ------------------------------------------------- | ------------------ |
+| Data-Only Onboard                                 | 1,000,000 DC ($10) |
+| Location Assert (incl elevation and antenna gain) | 500,000 DC ($5)    |
+| Transaction Fee                                   | ~0.1 Sol           |
 
+This HIP acknowledges that data transfer rewards and location metadata might not provide enough incentive for data-only Hotspot operators to fully onboard their hardware. Consequently, the Helium Network misses out on valuable data. On-chain knowledge of a Hotspot's location allows it to be included on maps, which the broader Helium community can rely on for their sensor deployments. Furthermore, the precise location of a Hotspot is advantageous for asset tracking applications that rely on multilateration.
 
-## Detailed Explanation
+During the time of writing, 1883 unonboarded and unasserted Hotspots were observed to be transferring data on the network. This HIP aims to enable these and future operators to onboard and assert their Hotspots without concern for the financial burden. The IoT Working Group believes that by adopting this proposal, Helium can enhance the IOT Network's robustness and utility while fostering a more vibrant and inclusive community of developers and users.
 
-- **Introduce and explain new concepts:**
-    - This HIP will reduce the cost of onboarding data-only hotspots from 1,000,000 DC ($10) to 500,000 DC ($5). After successful onboarding, each data-only hotspot will automatically receive one free location assertion every 30 epochs, requiring no Data Credits (DC). This initial assertion per period is free. Should further assertions be desired within the same period, they will be charged at 100,000 DC ($1) each.
+## Stakeholders
 
-- **How the proposal would be implemented:**
-    - Users will use the Helium Wallet App, Maker App, or CLI Wallet to initiate location assertions. The system will check if a free assertion is available; if not, a fee will apply. The onboarding process remains unchanged but at a reduced cost. Solana fees will still apply for both onboarding and assertions.
+Existing and future operators of data-only Hotspots will benefit from reduced costs or the ability to update their data-only Hotspot's location for a lower fee.
+
+## Detailed Explanation
 
+This HIP will reduce the cost of onboarding a data-only Hotspot from 1,000,000 DC ($10) to 50,000 DC ($0.50).  
+The cost of a location assertion will be reduced from 500,000 DC ($5) to 50,000 DC ($0.50).  
+SOL fees remain unaffected.
```

Comment:
```suggestion
This HIP will reduce the cost of onboarding a data-only Hotspot from 1,000,000 DC ($10) to 50,000 DC ($0.50). The cost of a data-only Hotspot's location assertion will be reduced from 500,000 DC ($5) to 50,000 DC ($0.50). SOL fees remain unaffected.
```

Reply:
flows a paragraph not halfway as a paragraph or a bulleted list. Can go the other way too, @jthiller.